src/scene/gsplat/gsplat-data.js

```diff
@@ -88,6 +88,10 @@ class GSplatData {
         if (!this.isCompressed && performZScale) {
             mat4.setScale(-1, -1, 1);
             this.transform(mat4);
+
+            // reorder uncompressed splats in morton order for better memory access
+            // efficiency during rendering
+            this.reorderData();
```

Comment:
should this only happen when `performZScale`?

Reply:
yeah right. i didn't want to but am forced to add a reorder option. thanks!